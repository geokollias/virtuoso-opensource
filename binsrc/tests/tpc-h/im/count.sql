select count(*) from REGION_F;
select count(*) from LINEITEM_F;
select count(*) from ORDERS_F;
select count(*) from PARTSUPP_F;
select count(*) from CUSTOMER_F;
select count(*) from SUPPLIER_F;
select count(*) from NATION_F;
select count(*) from PART_F;

