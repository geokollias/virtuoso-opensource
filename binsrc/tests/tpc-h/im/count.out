[?1034h+----------+
| count(*) |
+----------+
| 5        |
+----------+
+-----------+
| count(*)  |
+-----------+
| 600037902 |
+-----------+
+-----------+
| count(*)  |
+-----------+
| 150000000 |
+-----------+
+----------+
| count(*) |
+----------+
| 80000000 |
+----------+
+----------+
| count(*) |
+----------+
| 15000000 |
+----------+
+----------+
| count(*) |
+----------+
| 1000000  |
+----------+
+----------+
| count(*) |
+----------+
| 25       |
+----------+
+----------+
| count(*) |
+----------+
| 20000000 |
+----------+
